kernel/test/unit/catalog.saga.test.tsx

```diff
@@ -6,20 +6,20 @@ import { baseCatalogsLoaded, getExclusiveCatalog, getPlatformCatalog } from 'sha
 import { ensureRenderer } from 'shared/renderer/sagas'
 import { throwError } from 'redux-saga-test-plan/providers'
 
-const wearableId1 = 'WearableId1'
+const wearableId1 = 'dcl://base-avatars/name'
```

Comment:
This change was done because now, when a legacyId is requested, it is translated into urn before being sent to the catalyst